src/providers/recordset.tsx

```diff
@@ -352,6 +358,19 @@ export default function RecordsetProvider({
           } else {
             removeURLLimitAlert();
           }
+        } else if (config.displayMode === RecordsetDisplayMode.FK_BULK) {
+          // TODO: how to get numberForms?
+          // const numberFormsAllowed = RECORDEDIT_MAX_ROWS - numberForms
```

Comment:
I need to think about this a little bit more. 

How should `numberForms` be made available in recordset provider when it is used as a modal in recordedit app?